app/models/gems_adoption.rb

```diff
@@ -0,0 +1,3 @@
+class GemsAdoption < ActiveRecord::Base
```

Comment:
Yes, i think we have to think of a better name for this model. This was a tentative name.